ci/checks/changelog.sh

```diff
@@ -1,7 +1,7 @@
 #!/bin/bash
-# Copyright (c) 2020, NVIDIA CORPORATION.
+# Copyright (c) 2020-2022, NVIDIA CORPORATION.
 #########################
-# cuML CHANGELOG Tester #
+# RAFT CHANGELOG Tester #
```

Comment:
This `ci/checks/changelog.sh` file can be removed entirely. We no longer do changelog checks for RAPIDS libraries since the changelog process is now automated.